### PR TITLE
Handle logged in user id

### PR DESCRIPTION
**Pull request by btry**
depends: https://github.com/glpi-project/glpi/pull/6392

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [ ] Docs have been added/updated

### References

Closes #N/A

### CODE REVIEW

File: src/Glpi/Api/Rest/Client.php [hunk 1]
```diff
@@ -78,6 +81,21 @@ public function setSessionToken($token) {
...
+    * Get the ID of the currently logged in user
+    *
+    * @return null|integer
+    */
+   public function getSessionUserId() {
+      if ($this->sessionToken === null) {
+         return null;
+      }
+      if ($this->loginUserId === null) {
+         $this->getFullSession();
```
Comment (btry) [inline src/Glpi/Api/Rest/Client.php:94]: For compatibility with GLPI < 9.5

File: src/Glpi/Api/Rest/Client.php [hunk 2]
```diff
@@ -182,7 +207,14 @@ public function request($method, $uri, array $options = []) {
     */
    public function getFullSession() {
       $response = $this->request('get', 'getFullSession');
-      return ['statusCode' => $response->getStatusCode(), 'body' => $response->getBody()->getContents()];
+      $response = [
+         'statusCode' => $response->getStatusCode(), 
+         'body' => $response->getBody()->getContents()
+      ];
+      if ($this->loginUserId === null && $response['statusCode'] === 200) {
+         $this->loginUserId = (int) json_decode($response['body'], true)['session']['glpiID'];
```
Comment (btry) [inline src/Glpi/Api/Rest/Client.php:215]: For compatibility with GLPI < 9.5

**Review by cedric-anne — APPROVED** [review]

**Review by trasher — APPROVED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -21,6 +21,8 @@ matrix:
   exclude:
     - php: 5.6
       env: GLPI_BRANCH=9.5/bugfixes
+    - php: 5.6
+      env: GLPI_BRANCH=master
```
Comment (trasher) [inline .travis.yml:25]: glpi master? you sure?

Comment (btry) [reply .travis.yml:25]: yes : the goal is to not run php 5.6 with glpi master

Comment (trasher) [reply .travis.yml:25]: Oh! I've read too fast, sorry :)

**Pull request merged**